Terminal.Gui/Views/ComboBox.cs

```diff
@@ -983,4 +983,14 @@ private void SetInitialProperties (ComboBox container, bool hideDropdownListOnCl
             AddCommand (Command.LineUp, () => _container.MoveUpList ());
         }
     }
+
+    /// <inheritdoc />
+    public bool EnableForDesign ()
+    {
+        var source = new ObservableCollection<string> (["Combo Item 1", "Combo Item two", "Combo Item Quattro", "Last Combo Item"]);
+        SetSource (source);
```

Comment:
Also not relevant to this PR, and mentioned in an actually related PR, but just noting:

The SetSource method needs a bit more tweaking, precisely because of situations like this. The collection can be replaced wholesale, by a reference to a new collection, but anyone subscribed to the old collection's events won't ever know about it.

Easy to fix, though there are multiple ways to do so.

Again, not directly relevant for this PR. Just keeping it visible.